grails-app/assets/javascripts/streama/templates/admin-reports.tpl.htm

```diff
@@ -0,0 +1,106 @@
+<h1>Reports</h1>
+
+
+
+<p>Here you can see the reports users have made.</p>
+
+<div class="row">
+  <div class="col-md-12">
+
+  <div class="btn-group">
+    <button class="btn btn-default btn-sm pull-left show-reports-button"
+            ng-class="{'btn-primary': vm.listFilter == 'all'}"
+            ng-click="vm.refreshList('all')">
+      All
+    </button>
+
+    <button class="btn btn-default btn-sm pull-left show-reports-button"
+            ng-class="{'btn-primary': vm.listFilter === 'unresolved'}"
+            ng-click="vm.refreshList('unresolved')">
+      Only Unresolved Reports
+    </button>
+
+    <button class="btn btn-default btn-sm pull-left show-reports-button"
+            ng-class="{'btn-primary': vm.listFilter === 'resolved'}"
+            ng-click="vm.refreshList('resolved')">
+      Only Resolved Reports
+    </button>
+  </div>
+
+  <button class="btn btn-success btn-sm pull-right resolve-reports-button" ng-click="vm.resolveMultiple()">
+    Resolve Multiple Reports
+  </button>
+
+  </div>
+</div>
+
+<div class="report-sorter">
+  {{'DASHBOARD.SORT' | translate}}
+  <video-sort-order-dropdown  ng-model="vm.sortAndOrderBy" ng-change="vm.refreshList()"  dropdown-type="'report'"></video-sort-order-dropdown>
+</div>
+
+<div >
+  <table class="table table-striped table-reports">
+    <thead>
+    <tr>
+      <th></th>
+      <th>Report ID</th>
+      <th>Reported On</th>
+      <th>Last Updated</th>
+      <th>User</th>
+      <th>Play</th>
+      <th>Edit Page</th>
+      <th></th>
+    </tr>
+    </thead>
+    <tr ng-repeat="report in vm.reports | filter: vm.showReports">
+
+      <td>
+        <input type="checkbox" ng-if="report.resolved == false" ng-click="vm.addOrRemoveFromSelection($event, report)">
+      </td>
+
+      <td>
+        {{report.id}}
+      </td>
+
+      <td>
+        {{report.dateCreated | date : 'short'}}
+      </td>
+
+      <td>
+        {{report.lastUpdated | date : 'short'}}
+      </td>
+
+      <td>
+        {{report.userName}}
+      </td>
+
+      <td >
+        <a class="ion-android-arrow-dropright-circle"
+           style="font-size: 30px"
+           ui-sref="player({videoId: report.videoId})"></a>
+      </td>
+      <td>
+        <a ng-if="report.showId" ui-sref="admin.show({showId: report.showId})">{{report.videoTitle}}</a>
+        <a ng-if="!report.showId" ui-sref="admin.movie({movieId: report.videoId})">{{report.videoTitle}}</a>
+        <p ng-if="report.episodeString">
+          ({{report.episodeString}})
+        </p>
+      </td>
+
+
+      <td>
+        <button ng-if="report.resolved == false" title="Resolve report" class="btn btn-xs btn-success" ng-click="vm.resolve(report)"><i class="ion-checkmark"></i></button>
+        <button ng-if="report.resolved == true" title="Unesolve report" class="btn btn-xs btn-danger" ng-click="vm.unresolve(report)"><i class="ion-close"></i></button>
```

Comment:
i actually really like the styling of these buttons, but here is my problem. 
![image](https://user-images.githubusercontent.com/936076/34442637-beaf606a-ecc3-11e7-9e5e-a6ad83e7c6a8.png)

Imagine an untrained eye. The green check, without knowing that its a button, almost looks like its telling the user "all is good here", where the rest X sort of communicates "something is bad here". wheras in reality, you intended it to mean the exact opposite, the red X means, that it has been resolved. 

here is my suggestion: make a column in this table specifically for showing the report status, i.e. something red for "still a problem" and something green for "all good here, resolved already". And then add a neutral action button in the very right with a text instead of icon, a text that says either "mark as resolved" or "mark as unresolved" or similar. That way the UI is a lot less "clean", but its also less confusing.

Reply:
I agree, I actually had the same exact idea you are proposing, but forgot to add it in, I was just gonna use the buttons I have but make them unclickable and put them in their own resolved column and add another column that reads resolve/unresolve based on whether it is currently resolved or not.

Reply:
Fixed now. Could not however figure out how to remove the default button hover and focus behaviour.

Reply:
Let me check and get back to you

Reply:
fixed